src/test/java/org/sonarsource/sonarlint/ls/connected/ProjectBindingManagerTests.java

```diff
@@ -796,12 +641,11 @@ private WorkspaceFolderWrapper mockFileInABoundWorkspaceFolder() {
     return folder;
   }
 
-  private WorkspaceFolderWrapper mockFileInABoundWorkspaceFolder2() {
+  private void mockFileInABoundWorkspaceFolder2() {
```

Comment:
Methods with similar names works differently now. A bit confusing. Maybe it's not a bad thing to ignore return value.

Reply:
OK I will keep it for consistency